contributing/deployment-tools.txt

```diff
@@ -0,0 +1,113 @@
+OME deployment tools
+====================
+
+This section describes deployment tools supported by the OME team.
+It is primarily designed for the core OME developers who want to
+bring new or upgrade existing prerequisites. The following steps explain
+the connections between basic repositories and the testing workflow.
+
+Prerequisites locations
+-----------------------
+
+List of OME prerequisites is stored in multiple git repositories, each of 
+which is available from several locations.
+
+Infrastructure (Experimental)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Infrastructure is provided to simplify deployment using 
+`Ansible Tower<https://www.ansible.com>`.
+
+The Infrastructure repository is available from:
+
+-  https://github.com/openmicroscopy/infrastructure
+
+
+OMERO-install
+^^^^^^^^^^^^^
+
+OMERO installation scripts are provided to help new users with installing
+OMERO.server for the first time on a clean system, and can be used as
+the basis for more advanced configurations.
+
+The OMERO-install repository is available from:
+
+-  https://github.com/ome/omero-install
+
+
+Testing workflow
+----------------
+
+Devspace (Experimental)
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Continuous integration tools managed by Jenkins CI providing
+an automation framework that runs repeated jobs. The default deployment
+initializes a Jenkins CI master with a predefined set of jobs.
+
+The Devspace repository is available from:
+
+-  https://github.com/openmicroscopy/devspace
+
+Devspace Dockerfiles uses common devslave image. The Devslave repository
+is available from:
+
+-  https://github.com/openmicroscopy/devslave-c7-docker
+
+
+Continuous Delivery (Production)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Production Continuous Delivery (CD) platform managed by
+`Jenkins <http://jenkins-ci.org>`_ available on http://ci.openmicroscopy.org.
+More details about CI-master available on :doc:`continuous-integration`
+
+
+How to add new/upgrade prerequisites
+------------------------------------
+
+When the OME platform requires a new set of prerequisites all the above
```

Comment:
Agreed (although the test coverage is currently minimal). I guess most of our current examples are about OMERO but I could see e.g. the Java 6->Java7 being treated in the way you describe